ps_mbo.php

```diff
@@ -121,8 +121,6 @@ public function __construct()
         if ($this->active) {
             $this->bootHooks();
         }
-
-        $this->loadEnv();
```

Comment:
This method is called only here I think, we can remove it 🔥

Reply:
👍🏾 Yes it can be removed indeed